src/web/app/pages-instructor/instructor-audit-logs-page/instructor-audit-logs-page.component.spec.ts

```diff
@@ -45,13 +45,33 @@ describe('InstructorAuditLogsPageComponent', () => {
     timeZone: 'Asia/Singapore',
     creationTimestamp: 0,
     deletionTimestamp: 0,
+    privileges: {
+      canModifyCourse: true,
+      canModifySession: true,
+      canModifyStudent: true,
+      canModifyInstructor: true,
+      canViewStudentInSections: true,
+      canModifySessionCommentsInSections: true,
+      canViewSessionInSections: true,
+      canSubmitSessionInSections: true,
+    },
   };
   const testCourse2: Course = {
     courseId: 'MA1234',
     courseName: 'MA1234',
     timeZone: 'Asia/Singapore',
     creationTimestamp: 0,
     deletionTimestamp: 0,
+    privileges: {
+      canModifyCourse: true,
+      canModifySession: true,
+      canModifyStudent: true,
+      canModifyInstructor: true,
+      canViewStudentInSections: true,
+      canModifySessionCommentsInSections: true,
+      canViewSessionInSections: true,
+      canSubmitSessionInSections: true,
+    },
   };
```

Comment:
Should we also add a test course without `CAN_MODIFY_STUDENT` privileges, to ensure that it doesn't show up in the dropdown?